src/bundle/Controller/AssetController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUiBundle\Controller;
+
+use eZ\Publish\Core\FieldType\Image\Value as ImageValue;
+use eZ\Publish\Core\FieldType\ImageAsset\Mapper as ImageAssetMapper;
+use EzSystems\EzPlatformAdminUi\Form\Data\Asset\AssetUploadData;
+use EzSystems\EzPlatformAdminUi\Form\Factory\FormFactory;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class AssetController extends Controller
```

Comment:
I doubt it's a good idea to keep the Controller in AdminUI - what if we need this fieldtype to be available for frontend editing?